IFC4x3/Properties/e/EfficiencyPoint_2cbW2T9CPEk9B$EXGpp6o5/Documentation.md

```diff
@@ -0,0 +1 @@
+The point at which the fan operates, e.g. 50%.
```

Comment:
Proposal since the value is 0.0-1.0 or we write 50% and explain that the value will be 0.5.
```suggestion
The point at which the fan operates, e.g. 0.5.
```